packages/transition-common/src/services/nodes/NodeGeographyUtils.ts

```diff
@@ -35,6 +35,49 @@ const placesInBirdRadiusMeters = (node: Node, placeCollection: PlaceCollection,
     return placesInBirdRadius;
 };
 
+/**
+ * Find the touching point between two lines.
+ * It will return point only if the first line as a common point
+ * with second line and both are extremities.
+ * @param line1: a GeoJSON.Feature<GeoJSON.LineString>
+ * @param line2: a GeoJSON.Feature<GeoJSON.LineString>
+ * @returns a GeoJSON.Feature<GeoJSON.Point> or undefined
+ */
+export const findTouchingPoint = (
```

Comment:
Est-ce que cette fonction a besoin d'être exportée? Si c'est pour bypasser un bug dans turf, ce serait mieux de la laisser interne, elle sera plus facile à enlever après puisqu'elle ne fera pas partie de l'API et donc pas utilisé par d'autres packages

Reply:
To be able to unit test it